report_pre_thesis/src/sections/related_work.tex

```diff
@@ -1,14 +1,43 @@
 \section{Related work}\label{sec:related_works}
-% The Literature Review provides an overview of existing work related to your study.
-% - Purpose: To show how your work fits into the existing body of knowledge, identify gaps, and justify your research.
-% - Subsections: May be organized by methodologies, chronological developments, or thematic topics.
-% - Best Practices: Maintain a critical perspective, identifying not just what other works have found but also their limitations.
-
-% Introduction to the Field
-% - Overview of the research area
-% Existing Solutions
-% - Discussion of current methods, algorithms, or systems
-% Gaps in Current Research
-% - Identified shortcomings in existing work
-% Theoretical Frameworks
-% - Any theories or models that are foundational to the research
\ No newline at end of file
+Quantitative analysis of LIBS spectra is a well-studied problem.
```

Comment:
We talked about this one today. Left intentionally or?

As we spoke about, it might make more sense to say this in the intro, as you do. https://github.com/chhoumann/thesis-chemcam/pull/21/files